### ISSUE TITLE
Batch generation possible?

### ISSUE THREAD

**Issue by cblinde** [1aienthusiast/audiocraft-infinity-webui#12]
Is there a way to genrate more than 1 track a one using the same prompt an settings? Would be nice to have if not available at present. Thanks in advance

**Comment by jeffdbeats** [comment]
Yes, I also wanted this feature, so with the help of ChatGPT I was able to create a 'webuibatch.py'  file, it is basically the same thing as webui.py but it has a "batch mode" checkbox and then an "infinite" checkbox to just keep it running... so far it is working for me in infinite mode... also I made it so it doesn't play the music when in batch mode, since you are just letting it run making music... here it is: https://github.com/jeffdbeats/audiocraft-infinity-webui/blob/main/webuibatch.py (I am not an expert coder so it may not be super perfect, but it is working for my needs!) 
![image](https://github.com/1aienthusiast/audiocraft-infinity-webui/assets/13652893/fa73d028-8cab-4b3e-83bb-ec5b612f4f5a)
